### PR TITLE
CSS: Fixed the Bootstrap license block

### PR DESCRIPTION
**Pull request by LaurentGoderre**

### CODE REVIEW

**Comment by LaurentGoderre** [comment]
This will propagate to all the themes. The only thing is that there will be a duplicate glyphicon in the themes that we will need to get rid of.

**Comment by LaurentGoderre** [comment]
Don't merge yet!

**Comment by LaurentGoderre** [comment]
Ok, there you go, ready

**Comment by LaurentGoderre** [comment]
I believe this is ready to go

File: src/base/_bootstrap.scss [hunk 1]
```diff
@@ -3,5 +3,14 @@
 $icon-font-path: "#{$wb-core-path}/fonts/";
 
 
+/* TODO: Remove this block when including bootstrap 3.3.5 */
```
Comment (pjackson28) [inline src/base/_bootstrap.scss]: Why the TODO? What changes in 3.3.5?

Comment (LaurentGoderre) [inline src/base/_bootstrap.scss]: They include this block for us in the latest version while they do not in the one we use right now

**Comment by LaurentGoderre** [comment]
This is good to go!

**Comment by LaurentGoderre** [comment]
@MaximMartel can you go through all the themes and remove the following code in the CSS please?

```
/*!
 * GLYPHICONS Halflings for Twitter Bootstrap by GLYPHICONS.com | Licensed under http://www.apache.org/licenses/LICENSE-2.0
 */
```

**Comment by MaximMartel** [comment]
#1033

**Pull request merged**